src/platforms/java/common/legacy/spring/index.mdx

```diff
@@ -18,16 +18,16 @@ The source can be found [on GitHub](https://github.com/getsentry/sentry-java/tre
 <dependency>
     <groupId>io.sentry</groupId>
     <artifactId>sentry-spring</artifactId>
-    <version>{{ packages.version('sentry.java', '1.7.30') }}</version>
+    <version>1.7.30</version>
```

Comment:
Should we delete this page altogether?

We've released a new `sentry-spring` and it has top level docs.